README.md

```diff
@@ -21,7 +21,7 @@ Please feel free to submit PRs for any issues, errors, and/or additional functio
     * `fn cf a gilda random_channel_id <some_id>`
     * `fn cf a gilda webhook_url <some_url>`
     * if you don't want to post to url `fn cf a gilda is_local True`
-1. deploy locally `fn --verbose deploy --app gilda --all --local`
+1. deploy locally `fn --verbose deploy --app gilda --all --local --no-bump`
```

Comment:
Suggestion: can we update the readme (can be in another commit) that links to docs for the various docs for libraries and tools we use? Eg, I don't even know how to install this `fn` magic 😅 but I know it isn't `pip install fn`, that'll get you something totally different.